components/Clubroom.js

```diff
@@ -1,7 +1,37 @@
 import React from "react";
+import { Row, Col } from "react-bootstrap";
 
 const Clubroom = () => {
-	return <div>Clubroom</div>;
+	return (
+		<Row className='flex justify-between items-center bg-acm-blue rounded-2xl mx-auto w-2/3 px-4 pt-5 pb-4 mt-4'>
+			<Col sm={6} className='font-lexend text-acm-black font-medium'>
```

Comment:
`font-medium` isn't a thing I'm pretty sure, it should just be `font-base` check the docs on it, I remember using medium and it not working